internal/consensus/common_test.go

```diff
@@ -955,6 +949,7 @@ func makeGenesisState(ctx context.Context, t *testing.T, cfg *config.Config, arg
 		args.Time = time.Now()
 	}
 	genDoc, privVals := factory.RandGenesisDoc(cfg, args.Validators, 1, args.Params)
+	genDoc.GenesisTime = args.Time
```

Comment:
unfortunately, it is impossible. `genesisStateArgs` is not an exportable (internal) struct created for a test reason.
I see 3 options  
1. add a genesis-time field in an argument list (the same as in upstream), to do so, need to support it everywhere we use `RandGenesisDoc`
2. change the last type of an argument from `types.ConsensusParams` to a custom one with genesis time, the same as the previous it affects the code where we already use this function 
3. leave it as it is, since it is an exceptional case, where we have to change the genesis time from default (time.Now())